internal/pgbackrest/config.go

```diff
@@ -393,12 +393,14 @@ func reloadCommand(name string) []string {
 	// volume and configuration file. When either changes, signal pgBackRest
 	// and print the observed timestamp.
 	//
-	// We send SIGTERM because the TLS server in pgBackRest 2.36 must be
-	// restarted to change. We filter by parent process to ignore the forked
-	// connection handlers. Their parent process is one because they are
-	// detached/orphaned from the server. The server parent process is zero
-	// because it is started by Kubernetes.
-	//
+	// We send SIGHUP because this allows the TLS server configuration to be
+	// reloaded starting in pgBackRest 2.37. We filter by parent process to ignore
+	// the forked connection handlers.
+	// - https://github.com/pgbackrest/pgbackrest/commit/7b3ea883c7c010aafbeb14d150d073a113b703e4
+
+	// Their parent process is one because they are detached/orphaned from the server.
+	// The server parent process is zero because it is started by Kubernetes.
+
```

Comment:
⛏️ The connection handlers no longer detach.

```suggestion
	// reloaded starting in pgBackRest 2.37. We filter by parent process to
	// ignore forked connection handlers. The server's parent process is zero
	// because it is started by Kubernetes.
	// - https://github.com/pgbackrest/pgbackrest/commit/7b3ea883c7c010aafbeb
	//
```